src/chrome/content/rules/Fedora.xml

```diff
@@ -71,7 +72,7 @@
 	³ Secured by us
 
 -->
-<ruleset name="Fedora Project">
+<ruleset name="Fedora Project" default_off='failed ruleset test'>
 
```

Comment:
I'm guessing that blogs was the problematic rule. If so, why did you both remove blogs and disable the ruleset? If not, I need more information.